File: Java/src/main/java/org/codecop/dependencies/c/Receipt.java
package org.codecop.dependencies.c;

import java.util.Arrays;
import java.util.List;

public class Receipt {
    private Money amount;
    private Money tax;
    private Money total;

    public Money getAmount() {
        return amount;
    }

    public void setAmount(Money amount) {
        this.amount = amount;
    }

    public Money getTax() {
        return tax;
    }

    public void setTax(Money tax) {
        this.tax = tax;
    }

    public Money getTotal() {
        return total;
    }

    public void setTotal(Money total) {
        this.total = total;
    }

    public List<String> fomat() {
        return Arrays.asList( //
                "Receipt", //
                "=======", //
                "Item 1 ... " + amount.format(), //
                "Tax    ... " + tax.format(), //
                "----------------", //
                "Total  ... " + total.format() //
        );
    }
}

File: PHP/src/C/Receipt.php
<?php

namespace Org\Codecop\Dependencies\C;

class Receipt {
    private $amount;
    private $tax;
    private $total;

    public function getAmount(): Money {
        return $this->amount;
    }

    public function setAmount(Money $amount): void {
        $this->amount = $amount;
    }

    public function getTax(): Money {
        return $this->tax;
    }

    public function setTax(Money $tax): void {
        $this->tax = $tax;
    }

    public function getTotal(): Money {
        return $this->total;
    }

    public function setTotal(Money $total): void {
        $this->total = $total;
    }

    public function format(): array {
        return [ //
            "Receipt", //
            "=======", //
            "Item 1 ... " . $this->amount->format(), //
            "Tax    ... " . $this->tax->format(), //
            "----------------", //
            "Total  ... " . $this->total->format() //
        ];
    }
}
